https://github.com/sabotage-linux/gettext-tiny/issues/50

We're losing the battle of the bulge; someone really should make this use malloc.

--- gettext-tiny-0.3.1_git20191130/src/msgfmt.c.old	2020-01-14 08:03:11.000000000 +0000
+++ gettext-tiny-0.3.1_git20191130/src/msgfmt.c	2020-01-21 18:58:28.257709860 +0000
@@ -160,7 +160,7 @@
 
 int process(FILE *in, FILE *out, bool strict) {
 	struct mo_hdr mohdr = def_hdr;
-	char line[8192]; char *lp;
+	char line[12288]; char *lp;
 	size_t off, i;
 	enum po_error t;
 	char convbuf[32768];
